fvm/transaction_test.go

```diff
@@ -137,6 +138,7 @@ func TestSafetyCheck(t *testing.T) {
 
 		require.Contains(t, buffer.String(), "programs")
 		require.Contains(t, buffer.String(), "codes")
+		require.Equal(t, int(context.MaxNumOfTxRetries), proc.Retried)
```

Comment:
I think we would ultimately need one more test - one that fails TX once, so we retry, but not maximum number of times.

Reply:
yeah but was not sure how to do that, we might need to mocked runtime I guess